### PR TITLE
Add GitHub build, test, and lint actions

### PR DESCRIPTION
**Pull request by efoxepstein**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Lint
```
Comment (yunyeng) [inline .github/workflows/lint.yml:1]: Does this need License header as well?

**Review by yunyeng — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (efoxepstein) [reply .github/workflows/lint.yml:1]: I believe that's done in the last few lines of the file

**Review by SanjayVas — APPROVED** [review]
> Reviewed 2 of 2 files at r1, 8 of 9 files at r2, 1 of 1 files at r3, 18 of 18 files at r4.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/4)* status: all files reviewed, 2 unresolved discussions (waiting on @efoxepstein, @kunyuchen, @stevenwarejones, and @yunyeng)

___
*[.github/workflows/build-test.yml, line 34 at r4](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/4#-MZ0DJEM7sPGZMXij0na:-MZ0DJEN0-9rpqjkCWZI:b-jr55dm) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7435f701a112a7c56a4347d994f2db7f32ecf4b/.github/workflows/build-test.yml#L34)):*
> ```
> v1.3.0
> ```
nit: Technically the latest `actions` release is v1.4.0, but that didn't change anything for this action.
___
*[.github/workflows/lint.yml, line 1 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/4#-MZ0C_hnl9E4hY21xsvr-r1-1:-MZ0DaQ68RjJJwMNc8HV:b-5z8o32) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/416a6cbe6694b4521cfa50a77f05635bdfda4628/.github/workflows/lint.yml#L1)):*
<details><summary><i>Previously, efoxepstein (Eli Fox-Epstein) wrote…</i></summary><blockquote>

I believe that's done in the last few lines of the file
</blockquote></details>

I believe YAML files are considered configuration rather than source, so they don't need a license header.

What Eli is referring to is that this workflow configuration enforces license headers via `addlicense`.

**Review by stevenwarejones — APPROVED** [review]
I didn't look through the third_party stuff in great detail but the rest looks good. One comment on `node-modules`

File: .lintignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# File patterns that linters should ignore.
+#
+# This file must have valid syntax for both gitignore and hgignore.
+
+third_party/
+node_modules/
```
[the remark this answers is not in the document]
Comment (efoxepstein) [reply .lintignore:6]: Correct, just boilerplate.

**Review by yunyeng — APPROVED** [review]

**Pull request merged**